Add MAGIC_NUMBER for Python 3.5.2

kholia opened the pull request:
See https://bugs.python.org/issue27286 for some details. Python 3.5.2 uses "3351" as the magic number.

This change can also be verified [at this url](https://github.com/python/cpython/blob/master/Lib/importlib/_bootstrap_external.py).

I chose the string `MAGIC_352` over `MAGIC_3_5_2` to preserve the existing code indentation.

zrax commented:
Despite what the bug report claims, it appears this change was merged into 3.5.3, rather than 3.5.2.  I did a clean build of vanilla tarballs of both versions and only 3.5.3 generated .pyc files with the new magic number.  Also, I care about consistency (using '_') more than the pretty indentation :)

zrax commented on pyc_module.cpp:
```diff
@@ -130,6 +130,12 @@ void PycModule::setVersion(unsigned int magic)
         m_unicode = true;
         break;
 
+    case MAGIC_352:
```
This could be added inline with the `MAGIC_3_5` case above, since the case content is identical.

kholia replied:
I am using case fall-through in the latest commit and hopefully that is OK.

```
    case MAGIC_3_5:
        /* fall through */

    case MAGIC_3_5_3:
        m_maj = 3;
        m_min = 5;
        m_unicode = true;
        break;
```

kholia commented:
Thanks for reviewing this PR. In the latest commit, I have changed `MAGIC_352` to `MAGIC_3_5_3` to reflect the fact that the new magic number was really introduced in Python 3.5.3. I have also fixed the commit message.
